docker/test/integration/environment.py

```diff
@@ -11,20 +12,31 @@ def raise_exception(exception):
     raise exception
 
 
-@fixture
-def test_driver_fixture(context):
-    context.test = MiNiFi_integration_test(context)
-    yield context.test
-    logging.info("Integration test teardown...")
-    del context.test
+def integration_test_cleanup(test):
+    logging.info("Integration test cleanup...")
+    del test
 
 
 def before_scenario(context, scenario):
-    use_fixture(test_driver_fixture, context)
+    if "skip" in scenario.effective_tags:
+        scenario.skip("Marked with @skip")
+        return
+
+    logging.info("Integration test setup at {time:%H:%M:%S:%f}".format(time=datetime.datetime.now()))
```

Comment:
I think the colon between `%S` and `%f` is a typo, it should be a dot (also at line 34)

Reply:
Update in cdb014f306f614bfad5e6120f089ab821c904677